packages/tpetra/core/test/CrsMatrix/CrsMatrix_UnitTests.cpp

```diff
@@ -113,13 +113,13 @@ namespace { // (anonymous)
     {
       RCPMap map  = createContigMapWithNode<LO,GO,Node>(INVALID,numLocal,comm);
       MV mv(map,1);
-      zero = rcp( new MAT(map,0,Tpetra::TPETRA_DEFAULT_PROFILE_TYPE) );
+      zero = rcp( new MAT(map,0,TPETRA_DEFAULT_PROFILE_TYPE) );
```

Comment:
If you're using the default, you could just remove the argument and rely on the default value.

Reply:
True; I was just doing minimal editing to make the compiler happy.